### PR TITLE
PCQ-1880 remove autoscaling from loader and consolidation

### PR DESCRIPTION
**Pull request by priyankaVerma21**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/PCQ-1880

### Change description ###
Remove autoScaling from loader and consolidation

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: apps/pcq/pcq-frontend/prod.yaml [hunk 1]
```diff
@@ -7,7 +7,6 @@ spec:
     nodejs:
       ingressHost: equality-and-diversity.platform.hmcts.net
       autoscaling:
-        minReplicas: 2
         maxReplicas: 3
```
Comment (MartinYSpasov) [inline apps/pcq/pcq-frontend/prod.yaml:10]: What's the autoscaling rule here?

File: apps/pcq/pcq-backend/prod.yaml [hunk 2]
```diff
@@ -8,5 +8,4 @@ spec:
       spotInstances:
         enabled: false
       autoscaling:
-        minReplicas: 2
         maxReplicas: 3
```
Comment (MartinYSpasov) [inline apps/pcq/pcq-backend/prod.yaml:11]: What's the autoscaling rule here?

**Review by MartinYSpasov — CHANGES_REQUESTED** [review]

File: apps/pcq/pcq-backend/prod.yaml (hunk 2 above)
Comment (priyankaVerma21) [reply apps/pcq/pcq-backend/prod.yaml:11]: It has been enabled in values.yaml

File: apps/pcq/pcq-frontend/prod.yaml (hunk 1 above)
Comment (priyankaVerma21) [reply apps/pcq/pcq-frontend/prod.yaml:10]: It has been enabled in values.yaml

**Comment by priyankaVerma21** [comment]
Closing this PR . will do backend First

**Pull request closed without merging**